src/styles/day.scss

```diff
@@ -0,0 +1,28 @@
+@import './index.scss';
+
+.calendar__day {
+  width: $day-size;
```

Comment:
Ensure that the variable `$day-size` is defined in the `index.scss` file or another imported SCSS file. Otherwise, this will cause an error during compilation.